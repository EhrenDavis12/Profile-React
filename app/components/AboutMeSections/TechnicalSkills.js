import React from 'react';
import Banner from 'components/Banners/Banner';
import { PLeft } from 'components/SharedStyles/styles';
import Languages from 'components/Professional/Languages';

// const BGIcon = require('images/BGIcon.png');

function TechnicalSkills() {
  return (
    <PLeft>
      <Banner>
        <Languages />
      </Banner>
    </PLeft>
  );
}

export default TechnicalSkills;
