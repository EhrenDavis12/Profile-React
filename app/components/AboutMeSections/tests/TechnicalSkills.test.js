import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import TechnicalSkills from '../TechnicalSkills';

describe('<TechnicalSkills />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<TechnicalSkills />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
