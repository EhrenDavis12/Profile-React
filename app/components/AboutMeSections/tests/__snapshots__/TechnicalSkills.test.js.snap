// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TechnicalSkills /> should render the page 1`] = `
<styles__PLeft>
  <BannerSection>
    <TechnicalSkills />
  </BannerSection>
</styles__PLeft>
`;
